main.go

```diff
@@ -20,6 +22,16 @@ func failf(msg string, args ...interface{}) {
 	os.Exit(1)
 }
 
+func hasAnalyzeError(cmdOutput string) bool {
+	// example: error • Undefined class 'function' • lib/package.dart:3:1 • undefined_class
+	analyzeErrorPattern := regexp.MustCompile(`error.+\.dart.\s*`)
```

Comment:
The regexp matches dart + any character + any number of whitespace characters, please check that this is as intended.